docs/references/uvh5_memo.tex

```diff
@@ -746,7 +746,7 @@ \subsection{Writing strings in python2}
 with h5py.File('test_file.uvh5', 'w') as f:
     header = f.create_group('Header')
     # scalar dataset
-    header['scalar_string'] = np.string_('Hello world!')
```

Comment:
Are we sure we should change this python2 version?

Reply:
oh wow I actually missed that this was in python2. was on a rampage earlier i guess :laughing: 
I can confirm though, that `numpy.bytes_` exists in the python2 compatible version of numpy (version 1.16)